src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -145,19 +159,18 @@ bool emberAfGeneralCommissioningClusterCommissioningCompleteCallback(
     DeviceControlServer * server = &DeviceLayer::DeviceControlServer::DeviceControlSvr();
 
     /*
-     * Pass fabric and nodeId of commissioner to DeviceControlSvr.
+     * Pass nodeId of commissioner to DeviceControlSvr.
      * This allows device to send messages back to commissioner.
      * Once bindings are implemented, this may no longer be needed.
      */
     SessionHandle handle = commandObj->GetExchangeContext()->GetSessionHandle();
-    server->SetFabricIndex(handle->GetFabricIndex());
+
     server->SetPeerNodeId(handle->AsSecureSession()->GetPeerNodeId());
```

Comment:
Please make sure an issue is filed to remove this hack.  It's broken.

Reply:
Removed by https://github.com/project-chip/connectedhomeip/pull/15504